.gitlab-ci.yml

```diff
@@ -295,16 +295,13 @@ windows32:
     - /^pr-.*$/
 
 lint:
-  image: docker:git
   stage: test
-  script:
-    - apk add bash
-    - dev/lint-repository.sh
+  script: dev/lint-repository.sh
   dependencies: []
-  before_script: []
   variables:
-    # we need an unknown amount of history for per-commit linting
-    GIT_DEPTH: ""
+    GIT_DEPTH: "" # we need an unknown amount of history for per-commit linting
+    OPAM_SWITCH: edge
```

Comment:
IMHO using the `base` compiler could be a better idea here, it is going to be faster than the flambda build very likely; even if in theory they should be the same speed.

Reply:
changted